src/main/webapp/script.js

```diff
@@ -19,75 +19,69 @@ google.charts.load('current', {
 });
 
 function passQuery() {
-  const information = document.getElementById('more-info');
-  information.innerText = 'Please wait. Loading...';
```

Comment:
Is there a reason/did you mean to delete the part that adds the loading text? Not sure if you implemented a similar thing elsewhere, but my original thought was that showing the result div (or even just the more-info div) now would allow that loading text to be seen by the user. If there's no loading text, no need to make the div visible.

Reply:
Yeah, no I didn't mean to delete that, when I look at master I don't see these lines though, you might have to merge your branch into master first. I started this branch directly from master